### DISCUSSION TITLE
Enroute: iOS Version in public beta, request feedback

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#330] [Team Posts]
Dear all,

Simon Schneider has meanwhile ported "Enroute Flight Navigation" to iOS, and the app is now available in the Apple Store for a public beta test. We'd love for you to try it out and give us feedback. There are zillions of different devices that we cannot all test ourselves.

The installation is done in 2 steps.

Step 1: If you haven't already done so, you must install the "TestFlight" app first. This app is available in the Apple Store; it allows you to participate in the beta program: https://apps.apple.com/de/app/testflight/id899247664.

Step 2: After installing " TestFlight, " please open the following link in Safari browser to install Enroute Flight Navigation. https://testflight.apple.com/join/jqPSdGNX

Note: if you start the app "TestFlight" directly after step 1, you will be asked for an "invitation code." We do not use invitation codes, so please ignore the question and continue with step 2.

We are looking forward to hearing from you! Please forward the message to anyone who might be interested.

Best wishes,

Stefan.

**Comment by manolohpm** [comment]
Hello.

I'm testing it and everything looks fine. iPad 9th Gen here.

It detects which maps need to be download when first started, and connects with Xplane with no problem.

Like with the Android app, it looses connection from XPlane frequently and reconnects after 3-4 seconds.

Connected to an Android device (access point) looks to work fine too.

Tomorrow I'll test connection with Stratux.

Do you need anything else to be tested?

Best regards.

**Comment by kebekus** [comment]
@manolohpm Thank you for the feedback -- and please apologize for my late reply … these are quite stressful times at my job. I am glad that the app works for you. There is nothing specific that needs testing, but I would be interested to hear from you how the app works for you on a real flight!

I am closing this issue now, but your replies will still reach me.

Best wishes from Freiburg,

Stefan.
